tools/tool-env/install-git-hooks.ts

```diff
@@ -101,6 +100,6 @@ function removeMeteorInjectedHook(hookFile) {
 
   // Remove the line added to the file
   files.writeFile(hookFile,
-    files.readFile(hookFile, 'utf8').replace(METEOR_HOOK_LINE_REGEX, ''),
+    files.readFile(hookFile, 'utf8').toString().replace(METEOR_HOOK_LINE_REGEX, ''),
```

Comment:
Yeah, this is very true! I debated how to handle this with myself a bit.

I am pretty sure that if you read a file as `utf8` it actually always will return a string and not a buffer, but I wanted to respect the type param instead of overriding it, since the cast felt dirty, so I went with a simple `toString()` with the assumption that it's not really harmful even if it's probably unnecessary.